py/kubeflow/testing/cleanup_ci.py

```diff
@@ -881,6 +881,8 @@ def cleanup_clusters(args):
 
   expired = []
   unexpired = []
+  stopping = []
```

Comment:
could you help correct the log in LN 875?

Reply:
Done